### PR TITLE
[W4.11e][T13-3]Yingnan

### PR DESCRIPTION
**Pull request by NancyQuris**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -36,6 +36,24 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void hasNullObject() throws Exception {
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:41]: Avoid unnecessary line breaks.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -36,6 +36,24 @@ public void elementsAreUnique() throws Exception {
...
+
+        //one null object
+        assertTrue(Utils.isAnyNull((Object) null));
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:45]: Maybe add test case for a empty list as well?

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -36,6 +36,24 @@ public void elementsAreUnique() throws Exception {
...
+        //one not null object
+        assertFalse((Utils.isAnyNull("a")));
+
+        //list contains null object(s)
+        assertTrue(Utils.isAnyNull(null, null));
+        assertTrue(Utils.isAnyNull(null, "aa", "bb"));
+
+        //list of not null objects
+        assertFalse(Utils.isAnyNull("aaa", "bbb", "ccc"));
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:54]: Could try adding test cases for other types of list items, integers etc.

**Review by parvathy-sudhir-pillai — COMMENTED** [review]
Overall, good effort @NancyQuris. See my comments for suggestions. Please close this PR after reading comments.

**Comment by NancyQuris** [comment]
Thank you Ms Parvathy for the comment! I have read through them.
